acme/acme/challenges_test.py

```diff
@@ -94,7 +94,8 @@ def setUp(self):
         self.response = self.chall.response(KEY)
```

Comment:
I think we should also add simple unit tests for `to_partial_json` with the `keyAuthorization` to this file.

Reply:
Done.